projects.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "name": "LUG Website 2018-2019 Redesign",
+    "status": "active",
+    "description": "Redesign of the LUG Website for the modern era, and updated needs from it. ",
+    "smallImage": "https://lug.cs.uic.edu/wiki/lib/exe/fetch.php?media=20181204_145526.jpg",
+    "largeImage": "https://lug.cs.uic.edu/wiki/lib/exe/fetch.php?media=20181204_145526.jpg",
+    "wikiLink": "https://lug.cs.uic.edu/wiki/doku.php?id=webredesign2018",
+    "githubLink": "https://github.com/lugatuic/lug.cs.uic.edu-2018",
+    "startDate": "9/12/2018",
```

Comment:
While parsing this seems fine on my browser, I'm not 100% sure how this will parse in other regions (e.g. parsing `MM/DD/YYYY` or `DD/MM/YYYY`). Maybe we should agree on a date format? I don't know if backend should worry about this as there aren't any sorting/filtering features on the backend currently (@wilt00), but any format that the JavaScript method `new Date('DATE_STRING')` can parse works for frontend. For example, `Sep 12 2018` works as a date, while not being ambiguous about which is month or day.

Potentially useful: [MDN Date documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)

Reply:
Let's standardize on ISO8601